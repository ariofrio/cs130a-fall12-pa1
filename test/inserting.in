verbose
print

insert 12
print

insert 56
print

insert 899
print

insert -50
print

insert 0
print

insert -1 
print

insert 1 
print

lookup 12
lookup 56
lookup 899
lookup -50
lookup 0
lookup -1
lookup 1
