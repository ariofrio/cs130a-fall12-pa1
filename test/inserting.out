verbose mode enabled
0:0:-1 1:0:-1 2:0:-1 3:0:-1 4:0:-1 5:0:-1 6:0:-1 7:0:-1 8:0:-1 9:0:-1 10:0:-1 11:0:-1 12:0:-1 13:0:-1 14:0:-1 15:0:-1 16:0:-1 17:0:-1 18:0:-1 19:0:-1 20:0:-1 21:0:-1 22:0:-1 23:0:-1 24:0:-1 25:0:-1 26:0:-1 27:0:-1 28:0:-1 29:0:-1 30:0:-1 31:0:-1 32:0:-1 33:0:-1 34:0:-1 35:0:-1 36:0:-1 37:0:-1 38:0:-1 39:0:-1 40:0:-1 41:0:-1 42:0:-1

item 12 successfully inserted
0:0:-1 1:0:-1 2:0:-1 3:0:-1 4:0:-1 5:0:-1 6:0:-1 7:0:-1 8:0:-1 9:0:-1 10:0:-1 11:0:-1 12:12:0 13:0:-1 14:0:-1 15:0:-1 16:0:-1 17:0:-1 18:0:-1 19:0:-1 20:0:-1 21:0:-1 22:0:-1 23:0:-1 24:0:-1 25:0:-1 26:0:-1 27:0:-1 28:0:-1 29:0:-1 30:0:-1 31:0:-1 32:0:-1 33:0:-1 34:0:-1 35:0:-1 36:0:-1 37:0:-1 38:0:-1 39:0:-1 40:0:-1 41:0:-1 42:0:-1
0:12:12
item 56 successfully inserted
0:0:-1 1:0:-1 2:0:-1 3:0:-1 4:0:-1 5:0:-1 6:0:-1 7:0:-1 8:0:-1 9:0:-1 10:0:-1 11:0:-1 12:12:1 13:56:0 14:0:-1 15:0:-1 16:0:-1 17:0:-1 18:0:-1 19:0:-1 20:0:-1 21:0:-1 22:0:-1 23:0:-1 24:0:-1 25:0:-1 26:0:-1 27:0:-1 28:0:-1 29:0:-1 30:0:-1 31:0:-1 32:0:-1 33:0:-1 34:0:-1 35:0:-1 36:0:-1 37:0:-1 38:0:-1 39:0:-1 40:0:-1 41:0:-1 42:0:-1
0:56:13 1:12:12
item 899 successfully inserted
0:0:-1 1:0:-1 2:0:-1 3:0:-1 4:0:-1 5:0:-1 6:0:-1 7:0:-1 8:0:-1 9:0:-1 10:0:-1 11:0:-1 12:12:1 13:56:2 14:0:-1 15:0:-1 16:0:-1 17:0:-1 18:0:-1 19:0:-1 20:0:-1 21:0:-1 22:0:-1 23:0:-1 24:0:-1 25:0:-1 26:0:-1 27:0:-1 28:0:-1 29:0:-1 30:0:-1 31:0:-1 32:0:-1 33:0:-1 34:0:-1 35:0:-1 36:0:-1 37:0:-1 38:0:-1 39:899:0 40:0:-1 41:0:-1 42:0:-1
0:899:39 1:12:12 2:56:13
item -50 successfully inserted
0:0:-1 1:0:-1 2:0:-1 3:0:-1 4:0:-1 5:0:-1 6:0:-1 7:0:-1 8:0:-1 9:0:-1 10:0:-1 11:0:-1 12:12:1 13:56:2 14:0:-1 15:0:-1 16:0:-1 17:0:-1 18:0:-1 19:0:-1 20:0:-1 21:0:-1 22:0:-1 23:0:-1 24:0:-1 25:0:-1 26:0:-1 27:0:-1 28:0:-1 29:0:-1 30:0:-1 31:0:-1 32:0:-1 33:0:-1 34:0:-1 35:0:-1 36:-50:3 37:0:-1 38:0:-1 39:899:0 40:0:-1 41:0:-1 42:0:-1
0:899:39 1:12:12 2:56:13 3:-50:36
item 0 successfully inserted
0:0:4 1:0:-1 2:0:-1 3:0:-1 4:0:-1 5:0:-1 6:0:-1 7:0:-1 8:0:-1 9:0:-1 10:0:-1 11:0:-1 12:12:1 13:56:2 14:0:-1 15:0:-1 16:0:-1 17:0:-1 18:0:-1 19:0:-1 20:0:-1 21:0:-1 22:0:-1 23:0:-1 24:0:-1 25:0:-1 26:0:-1 27:0:-1 28:0:-1 29:0:-1 30:0:-1 31:0:-1 32:0:-1 33:0:-1 34:0:-1 35:0:-1 36:-50:3 37:0:-1 38:0:-1 39:899:0 40:0:-1 41:0:-1 42:0:-1
0:899:39 1:12:12 2:56:13 3:-50:36 4:0:0
item -1 successfully inserted
0:0:4 1:0:-1 2:0:-1 3:0:-1 4:0:-1 5:0:-1 6:0:-1 7:0:-1 8:0:-1 9:0:-1 10:0:-1 11:0:-1 12:12:1 13:56:2 14:0:-1 15:0:-1 16:0:-1 17:0:-1 18:0:-1 19:0:-1 20:0:-1 21:0:-1 22:0:-1 23:0:-1 24:0:-1 25:0:-1 26:0:-1 27:0:-1 28:0:-1 29:0:-1 30:0:-1 31:0:-1 32:0:-1 33:0:-1 34:0:-1 35:0:-1 36:-50:3 37:0:-1 38:0:-1 39:899:0 40:0:-1 41:0:-1 42:-1:5
0:899:39 1:12:12 2:56:13 3:-50:36 4:0:0 5:-1:42
item 1 successfully inserted
0:0:4 1:1:6 2:0:-1 3:0:-1 4:0:-1 5:0:-1 6:0:-1 7:0:-1 8:0:-1 9:0:-1 10:0:-1 11:0:-1 12:12:1 13:56:2 14:0:-1 15:0:-1 16:0:-1 17:0:-1 18:0:-1 19:0:-1 20:0:-1 21:0:-1 22:0:-1 23:0:-1 24:0:-1 25:0:-1 26:0:-1 27:0:-1 28:0:-1 29:0:-1 30:0:-1 31:0:-1 32:0:-1 33:0:-1 34:0:-1 35:0:-1 36:-50:3 37:0:-1 38:0:-1 39:899:0 40:0:-1 41:0:-1 42:-1:5
0:899:39 1:12:12 2:56:13 3:-50:36 4:0:0 5:-1:42 6:1:1
item 12 found
item 56 found
item 899 found
item -50 found
item 0 found
item -1 found
item 1 found
